tests/acceptance/30_generic_methods/command_execution_once.cf

```diff
@@ -0,0 +1,85 @@
+#######################################################
+#
+# Test the exit code of shell commands
+#
+#######################################################
+
+bundle common acc_path
+{
+  vars:
+    "root" string => getenv("NCF_TESTS_ACCEPTANCE", 1024);
+}
+
+body common control
+{
+      inputs => { "${acc_path.root}/default.cf.sub", "${acc_path.root}/default_ncf.cf.sub", "@{ncf_inputs.default_files}" };
+      bundlesequence  => { configuration, initialization, default("${this.promise_filename}") };
+      version => "1.0";
+}
+
+#######################################################
+
+bundle agent init
+{
+  methods:
+      "clean" usebundle => file_absent("${configuration.flag_file}");
+}
+
+#######################################################
+
+bundle agent test
+{
+  vars:
+    "ok_codes" string => "0, 2,4, 6";
+
+  methods:
+    "phA"  usebundle => command_execution_once("exit 0",  "${ok_codes}", "any", "testA");
+    "phAA" usebundle => command_execution_once("exit 0 ", "${ok_codes}", "any", "testA");
+
+    "phB"  usebundle => command_execution_once("exit 1",  "${ok_codes}", "any", "testB");
+    "phBB" usebundle => command_execution_once("exit 1 ", "${ok_codes}", "any", "testB");
+
+    "phC"  usebundle => command_execution_once("exit 2",  "${ok_codes}", "ok", "testC");
+    "phCC" usebundle => command_execution_once("exit 2 ", "${ok_codes}", "ok", "testC");
+
+    "phD"    usebundle => command_execution_once("exit 3",  "${ok_codes}", "ok", "testD");
+    "phDD"   usebundle => command_execution_once("exit 3 ", "${ok_codes}", "ok", "testD");
+    "phDDD"  usebundle => command_execution_once("exit 4",  "${ok_codes}", "ok", "testD");
+
+    "enable"  usebundle => set_dry_run_mode("true");
+    "phE"     usebundle => command_execution_once("exit 5",  "${ok_codes}", "ok", "testE");
```

Comment:
auditing execute_once is an error ? this is surprising, but i don't know what it should be

Reply:
This is the current behavior with other command methods, I think it's the best we can do.

Reply:
I'm adding a test for already executed command in audit mode

Reply:
done